### PR TITLE
fix: Update the minimum character required for the password

### LINKED ISSUE #1943: Error during signup while entering password
**Describe the bug**
 When we signup for the first time and we are asked to enter the password.
So, it says minimum character should be 6. And when we enter 6 digit password, it shows that password should be of minimum 8 characters. 

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by Ritesh-Ag**
Fixes #1943 

Checklist:

I have increased the minimum number of character for entering password to 8.

Changes: Changed the value of app:validateMinLength=8

### CODE REVIEW

File: fastlane/metadata/android/en-US/changelogs/16.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+🚀 Features and Enhancements
+- feat(ui): Add event name and time in dashboard
+- feat: Add 'About The App' section
+- feat: Add bio section in Edit Profile
+- feat: Option to delete profile picture
```
Comment (liveHarshit) [inline fastlane/metadata/android/en-US/changelogs/16.txt]: Why? Do not commit this file.

Comment (Ritesh-Ag) [reply fastlane/metadata/android/en-US/changelogs/16.txt]: I am so sorry. I didn't commited the file intentionally. I am new to git and github. So, that is a mistake.

Comment (Ritesh-Ag) [reply fastlane/metadata/android/en-US/changelogs/16.txt]: @liveHarshit I have deleted the file. Sorry for the mistake sir!

**Comment by Ritesh-Ag** [comment]
> 
> 
> Should have 1 commit only

@iamareebjamal Sir, can you please guide me how to remove other commits now?

**Comment by iamareebjamal** [comment]
Always create a new branch for each PR

**Comment by liveHarshit** [comment]
https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/todotxt/todo.txt-android">todotxt/todo.txt-android</a></strong></div><div>Official Todo.txt Android app for managing your todo.txt file stored in Dropbox. - todotxt/todo.txt-android</div></blockquote>

**Comment by Ritesh-Ag** [comment]
@iamareebjamal @liveHarshit Thank you so much sir for the guidance. What should i do now? Should i close this pull request and make another PR with one commit?

**Comment by liveHarshit** [comment]
> @iamareebjamal @liveHarshit Thank you so much sir for the guidance. What should i do now? Should i close this pull request and make another PR with one commit?

Squash the commits into one and force push on the same branch from which you made the PR. No need to create new PR.

**Comment by Ritesh-Ag** [comment]
@liveHarshit Thank you so much sir! Learned so many new things.

File: app/build.gradle [hunk 2]
```diff
@@ -88,8 +88,8 @@ android {
         resValue "string", "YOUTUBE_ID", "UCQprMsG-raCIMlBudm20iLQ"
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
```
Comment (ShridharGoel) [inline app/build.gradle]: Revert.

Comment (Ritesh-Ag) [reply app/build.gradle]: Okay!

**Comment by ShridharGoel** [comment]
@Ritesh-Ag You have deleted the build.gradle file instead of reverting the unnecessary changes.

**Comment by liveHarshit** [comment]
> @liveHarshit Thank you so much sir! Learned so many new things.

You have to struggle for the first time, afterward, you will get comfortable.

**Comment by liveHarshit** [comment]
Also, always rebase your branch with the latest commits on upstream before sending a PR.

**Comment by Ritesh-Ag** [comment]
@liveHarshit Okay sir!

**Comment by Ritesh-Ag** [comment]
@ShridharGoel My fault, i have deleted the file. Can you please tell me how to revert back that commit?

**Comment by iamareebjamal** [comment]
When you have done so many unrelated changes to the PR and your commit is just a few lines of code, it is just easier to reset everything, make the change and force push.

```
git fetch upstream development
git reset --hard origin/development
# Do your changes. Commit
git push origin <branch_name> -f
```

**Comment by Ritesh-Ag** [comment]
@ShridharGoel Sir, is it all right now?

**Comment by liveHarshit** [comment]
Please follow commit style guidelines https://chris.beams.io/posts/git-commit/

<blockquote><div><strong><a href="https://chris.beams.io/posts/git-commit/">How to Write a Git Commit Message</a></strong></div></blockquote>

**Comment by Ritesh-Ag** [comment]
@liveHarshit Okay sir, I will take care of that.

**Comment by ShridharGoel** [comment]
@Ritesh-Ag Update the commit message.

**Comment by Ritesh-Ag** [comment]
@ShridharGoel Okay! Done sir.

**Comment by iamareebjamal** [comment]
Still not following the guidelines. Just see the names of merged PRs

**Comment by Ritesh-Ag** [comment]
@iamareebjamal Sir, is it correct now?

**Comment by ShridharGoel** [comment]
@Ritesh-Ag Just add a space after `:`

**Comment by Ritesh-Ag** [comment]
@ShridharGoel Okay sir, Done!

**Pull request merged**